data_collection/other_integrations/web_log_analytics.rst

```diff
@@ -16,19 +16,23 @@ Sample command:
 
 .. code-block:: bash
 
-    import_logs.py --url=https://demo.piwik.pro --token-auth=XXXXXXXXXXXXXXXXXXXXXXXXXXXXX --enable-static --enable-bots --show-progress --idsite=X --recorders=2 --recorder-max-payload-size=50 sample.log
+    import_logs.py --url=https://demo.piwik.pro --client-id=*** --client-secret=*** --enable-static --enable-bots --show-progress --idsite=*** --recorders=2 --recorder-max-payload-size=50 sample.log
```

Comment:
Let's set `--recorder-max-payload-size` to 1 - otherwise `--enable-bulk-tracking` has to be used, which is not supported in new tracker.

Reply:
Actually `--recorder-max-payload-size` is ignored without `--enable-bulk-tracking` since GET requests don't have bulk option. We can remove that option from  example or add `--enable-bulk-tracking`.

Reply:
Let's just remove `--recorder-max-payload-size`.